### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by goosewobbler**
* adding `prettier` (same config as Frame), `lint-staged` & `husky`
* added `jest` and placeholder test

TODO:
- [x] Verify lint-staged is required for precommit hook
- [ ] Use `@framelabs/prettier-config` - requires it to be published
- [x] Rebase and run formatting again to account for recently merged PRs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +24,12 @@
     "@babel/plugin-transform-runtime": "7.18.2",
     "@babel/preset-react": "7.17.12",
     "html-webpack-plugin": "5.5.0",
+    "husky": "8.0.3",
```
Comment (mholtzman) [inline package.json:27]: do we need lint staged?

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mholtzman) [inline .prettierrc.json:1]: let's use the global frame labs prettier config instead

**Review by mholtzman — CHANGES_REQUESTED** [review]

File: .prettierrc.json (hunk 2 above)
Comment (goosewobbler) [reply .prettierrc.json:1]: I get 404 trying to install - can we publish this config?

File: package.json (hunk 1 above)
Comment (goosewobbler) [reply package.json:27]: Yes we do, speeds up the precommit hook - it's not such a big difference when only running prettier but once you add linting in there as well it's quite noticeable.  Also ideally we have the same basic tooling across all our repos.

**Pull request closed without merging**